doc/DEVELOPERS.rst

```diff
@@ -50,25 +50,11 @@ activate a venv::
 
     git clone https://github.com/rst2pdf/rst2pdf
     cd rst2pdf
-    python3 -m venv .venv
-    . .venv/bin/activate
+    uv sync --all-extras
 
-Ensure that setuptools and pip are up to date::
+If you don't have ``uv``, please see `the installation docs <https://docs.astral.sh/uv/getting-started/installation/>`_
 
-    pip install --upgrade setuptools pip
-
-Now you can install rst2pdf from this source code::
-
-    pip install -c requirements.txt -e .[aafiguresupport,mathsupport,plantumlsupport,rawhtmlsupport,sphinx,svgsupport,tests]
```

Comment:
I see that we didn't replace this command with anything. So when I edit the source code, how do I get that updated version used? Run `uv sync` again?

Reply:
I think yes, I answered my own question here :)